nitransforms/io/lta.py

```diff
@@ -120,14 +177,15 @@ def to_string(self):
             ('{:18.15e} ' * 4).format(*sa['m_L'][2]),
             ('{:18.15e} ' * 4).format(*sa['m_L'][3]),
             'src volume info',
-            self['src'].to_string(),
+            '%s' % self['src'],
```

Comment:
i think this should work
```suggestion
            str(self['src']),
```